extendeddatacrossword.go

```diff
@@ -139,6 +139,9 @@ func (eds *ExtendedDataSquare) solveCrosswordRow(
 	// Check that rebuilt shares matches appropriate root
 	err = eds.verifyAgainstRowRoots(rowRoots, uint(r), rebuiltShares)
 	if err != nil {
+		if byzErr, ok := err.(*ErrByzantineData); ok {
```

Comment:
You should either use `errors.Is` or `errors.As` according to golang: https://pkg.go.dev/errors

Reply:
I also wanted to comment on this, but as long as it expects a specific type in here to amend it - it's fine. Though I think the way we set shares in it is rather hacky, but don't have anything else to propose as an easy fix

Reply:
Done